dialogue-core/src/main/java/com/palantir/dialogue/core/DialogueChannel.java

```diff
@@ -186,15 +186,14 @@ public Builder channelFactory(ChannelFactory value) {
             return this;
         }
 
-        @VisibleForTesting
-        Builder random(Random value) {
-            builder.random(value);
+        public Builder retryScheduler(ScheduledExecutorService value) {
```

Comment:
no longer package private + visible-for-testing?

Reply:
Right so I had to make this public in order to access it from `com.palantir.dialogue.clients`. Given that it wasn't being used before, shall I actually just leave it as non-configurable?

Reply:
Might as well leave it simple for now, we can always expose it later